SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,32 @@
+header:
+  schema-version: 1.0.0
+  last-updated: '2024-03-01'
+  last-reviewed: '2024-03-01'
+  expiration-date: '2025-03-01T10:00:00.000Z'
+  project-url: https://github.com/devfile/devworkspace-operator
+  project-release: '0.26.0'
+  commit-hash: '067847d900c18a3fe0d47de920a9ce77af29e722'
+  license: 'https://github.com/devfile/devworkspace-operator/blob/main/LICENSE'
+project-lifecycle:
+  status: active
+  bug-fixes-only: false
+  core-maintainers:
+  - github:AObuchow
+  - github:dkwon17
+  release-cycle: https://github.com/devfile/devworkspace-operator/blob/main/docs/release/README.md
```

Comment:
Not entirely sure if this field is supposed to point to the release documentation or the release cadence? We usually release DWO upstream in advance of an Eclipse Che release, as Eclipse Che depends on DWO.